doc/api/inspector.md

```diff
@@ -11,86 +11,205 @@ inspector.
 
 It can be accessed using:
 
-```js
+To use the promise-based APIs:
```

Comment:
This paragraph doesn't really make sense with:

```
It can be accessed using:

To use the promise-based APIs:
```

Maybe rephrase this whole area?

Reply:
I thought it's easier removing them and add just an `or` there. What do you think? (just committed the change as well)